src/custom/state/price/reducer.ts

```diff
@@ -53,29 +53,60 @@ function initializeState(
   }
 }
 
+function getResetPrice(sellToken: string, buyToken: string, kind: string) {
+  return {
+    amount: null,
+    token: kind ? sellToken : buyToken
```

Comment:
Shouldn't we make this kind a boolean, and call it `isSellToken`?
Because it's a string and the check is misleading.

Or make `kind: 'sell' | 'buy'` to be more explicit.

Reply:
Good catch!! I think this is important! 

I wiill actually do a hotfix for this.

Reply:
no hotfix needed, will do for 0.16: https://github.com/gnosis/cowswap/pull/845